app/utils/misc.js

```diff
@@ -25,3 +25,14 @@ export function debounce(func, ...theArgs) {
     }
   };
 }
+
+/**
+ * Determine if data is multi-series or single series
+ *
+ * @param array data Data array
+ * @return bool True if multi-series, false if single-series
+ */
+export function dataIsMultiSeries(data) {
+  return (typeof data[0].key !== 'undefined'
```

Comment:
`&&` should be at the end of the line, not the beginning... 

I just realized I need to add that to an eslint rule...  so for now it's fine.